src/web/app/pages-instructor/instructor-session-result-page/instructor-session-no-response-panel.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import { Student } from '../../../types/api-output';
+
+/**
+ * Instructor sessions results page No Response Panel.
+ */
+@Component({
+  selector: 'tm-instructor-session-no-response-panel',
+  templateUrl: './instructor-session-no-response-panel.component.html',
+  styleUrls: ['./instructor-session-no-response-panel.component.scss'],
+})
+export class InstructorSessionNoResponsePanelComponent implements OnInit, OnChanges {
+
+  @Input() noResponseStudents: Student[] = [];
+  @Input() section: string = '';
```

Comment:
Consider using https://angular.io/guide/component-interaction#intercept-input-property-changes-with-a-setter

Reply:
I've tried that, but that the setter seems to only react to initialisation to the property, here we must act to the change of `NoResponseStudent` and `section`.